libnetwork/iptables/iptables.go

```diff
@@ -555,7 +546,7 @@ func (iptable IPTable) raw(args ...string) ([]byte, error) {
 	startTime := time.Now()
 	output, err := exec.Command(path, args...).CombinedOutput()
 	if err != nil {
-		return nil, fmt.Errorf("iptables failed: %s %v: %s (%s)", commandName, strings.Join(args, " "), output, err)
+		return nil, fmt.Errorf("%s failed: %s %v: %s (%s)", commandName, commandName, strings.Join(args, " "), output, err)
```

Comment:
I don't think it hurts since the second usage is effectively printing the command that was executed, and the output is essentially the same as before... But maybe cleaner is better.

Reply:
I find it quite bad too, but I can't come up with a good, easy-to-grep error prefix :sweat_smile: Do you have any suggestion? Should I just revert to the initial error message?

Reply:
I mean, with this change, it goes from something like:

```
iptables failed: /usr/bin/iptables args here: output (error)
```

to:

```
/usr/bin/iptables failed: /usr/bin/iptables args here: output (error)
```

(Which I don't think actually adds anything and makes it harder to find the error in the code than what it was before :see_no_evil:)